### DISCUSSION TITLE
Visual Programming with Blockly-inspired System

### DISCUSSION THREAD

**Discussion by AndrewJSchoen** [xyflow/xyflow#1994] [Show and tell]
First, kudos to the react-flow team on a great system. It has been incredibly helpful to getting a working system on my end, and your answers to my questions have been great as well. I do a bunch of work with visual programming systems, and created a configurable [blockly](https://developers.google.com/blockly/)-inspired programming interface. 
<img width="669" alt="Screen Shot 2022-03-21 at 10 12 15 AM" src="https://user-images.githubusercontent.com/5341396/159291491-0719ebf0-1984-4224-a61a-2785a7bf4133.png">

There are still some bugs to work out, and it appears there might be some issues on certain browsers yet, but it is coming along!
You can find the [repo here](https://github.com/Wisc-HCI/simple-vp).

**Comment by moklick** [comment]
Hey @AndrewJSchoen 

this looks awesome! Thanks for sharing :)
